src/_shared/components/CardActionButtonRow/CardActionButtonRow.tsx

```diff
@@ -36,7 +36,7 @@ interface CardActionButtonRowProps {
 }
 
 export const CardActionButtonRow: React.FC<CardActionButtonRowProps> = (
-  props
+  props,
```

Comment:
A whole heap of linting tweaks were needed for the app to run locally - strange, but ok.

Reply:
i just ran into this too! weird...